samples/go/README-ptbr.md

```diff
@@ -0,0 +1,54 @@
+# Lead Integration - Go
+
+[English version](README.md)
+
+Um simples projeto usando Go para demonstrar como receber leads do zap+.
+
+Por favor, confira nossa [documentação](https://developers.grupozap.com/) para mais informações.
+
+## Recursos Utilizados
+
+- [Go 1.16.4](https://golang.org/)
```

Comment:
just to keep info updated easily, you can omit the minor 💅 
```suggestion
- [Go 1.16](https://golang.org/)
```

Reply:
Updated